src/Type/Components/MapTypes.ts

```diff
@@ -0,0 +1,26 @@
+export interface MarkerData {
+    position: [number, number]
+    popupContent: string
+}
+
+export interface CustomSelectPositionMapProps {
+    onPositionChange: (lat: number, lng: number) => void
+    center?: [number, number]
+    zoomLevel?: number
+}
+
+export interface DisplayPositionProps {
+    map: any
+    onPositionChange: (lat: number, lng: number) => void
+}
+
+export interface DisplayMapProps {
+    center: [number, number]
```

Comment:
what is this a map ??

Reply:
I created this to develop a seperate component for displaying maps, reducing redundant code.

Reply:
all of the above can be moved to one type

type MarkerPosition = [number, number]